packages/eui/src/components/form/select/select.stories.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import {
+  disableStorybookControls,
+  enableFunctionToggleControls,
+} from '../../../../.storybook/utils';
+import { EuiIcon } from '../../icon';
+
+import { EuiSelect, EuiSelectProps } from './select';
+
+const meta: Meta<EuiSelectProps> = {
+  title: 'Forms/EuiSelect',
+  component: EuiSelect,
+  argTypes: {
+    append: {
+      control: 'radio',
+      options: [undefined, 'icon', 'text'],
+      mapping: {
+        icon: <EuiIcon type="faceHappy" />,
+        text: 'Appended',
+        undefined: undefined,
+      },
+    },
+    prepend: {
+      control: 'radio',
+      options: [undefined, 'icon', 'text'],
+      mapping: {
+        icon: <EuiIcon type="faceHappy" />,
+        text: 'Prepended',
+        undefined: undefined,
+      },
+    },
+  },
+  args: {
+    fullWidth: false,
+    isLoading: false,
+    hasNoInitialSelection: false,
+    compressed: false,
+    // set up for easier testing/QA
+    isInvalid: false,
+    disabled: false,
+    id: '',
+    name: '',
+  },
+};
+// adding onChange for visibility
+enableFunctionToggleControls(meta, ['onChange', 'onMouseUp']);
```

Comment:
If we've excluded `onMouseUp`, do we still need it in this array?
```suggestion
enableFunctionToggleControls(meta, ['onChange']);
```